junit-platform-console/src/main/java/org/junit/platform/console/options/Theme.java

```diff
@@ -79,61 +81,60 @@ public static Theme valueOf(Charset charset) {
 		this.blank = new String(new char[vertical().length()]).replace('\0', ' ');
 	}
 
-	public String root() {
+	public final String root() {
 		return tiles[0];
 	}
 
-	public String vertical() {
+	public final String vertical() {
 		return tiles[1];
 	}
 
-	public String blank() {
+	public final String blank() {
 		return blank;
 	}
 
-	public String entry() {
+	public final String entry() {
 		return tiles[2];
 	}
 
-	public String end() {
+	public final String end() {
 		return tiles[3];
 	}
 
-	public String successful() {
+	public final String successful() {
 		return tiles[4];
 	}
 
-	public String aborted() {
+	public final String aborted() {
 		return tiles[5];
 	}
 
-	public String failed() {
+	public final String failed() {
 		return tiles[6];
 	}
 
-	public String skipped() {
+	public final String skipped() {
 		return tiles[7];
 	}
 
-	public String status(TestExecutionResult result) {
+	public final String status(TestExecutionResult result) {
```

Comment:
Okay.

Just to let you know that until the corresponding "Polishing" PR is merged (which I'll work on soon), Travis CI won't go green, because I made these `final` changes in response to the following compilation error reported by error-prone:

```
C:\Users\Jonathan\dev\Java\IntelliJ Projects\junit5\junit-platform-console\src\main\java\org\junit\platform\console\options\Theme.java:81: warning: [ConstructorInvokesOverridable] Constructors should not invoke overridable methods.
		this.blank = new String(new char[vertical().length()]).replace('\0', ' ');
		                                         ^
    (see http://errorprone.info/bugpattern/ConstructorInvokesOverridable)
```